apps/portal/components/containers/dashboardLayout/elements/SidebarMenuItems/Favorite.tsx

```
@@ -0,0 +1,114 @@
+}: AccordionItemProps) {
+  const [isOpen, setIsOpen] = useState(defaultOpen);
+
+  const handleToggle = (e: React.MouseEvent) => {
+    e.stopPropagation();
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <div>
+      <div className=" tw-flex tw-items-center tw-rounded-xl  tw-bg-[#383c44]/10 tw-w-[270px] tw-w-full tw-px-4 tw-py-4 tw-pl-[38px]">
+        <div onClick={handleToggle} className="tw-cursor-pointer tw-mr-2">
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)